common/device-model.cpp

```diff
@@ -1457,6 +1472,7 @@ namespace rs2
         }
 
         _calib_model.update(window, error_message);
+        _dds_model.render_dds_config_window(window , error_message);
```

Comment:
Will this work id we build with DDS off?

Reply:
I'll add a check to see if the device supports DDS before using this function. 
But in general, yes the functionality works with DDS off because the function already checks whether the window should be open. If the device doesn't support DDS, the window is closed by default.